src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ]  Please work on the division of a number with zero let us say `9/0` normally you have to get error in the screen but currently you are getting this

![Screenshot from 2021-09-15 20-49-51](https://user-images.githubusercontent.com/22434660/133462233-da92983f-72a2-4763-b127-c37c5a3d7a3f.png)